bases/node20.json

```diff
@@ -11,6 +11,6 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node16"
```

Comment:
Should this be `nodenext` instead?

Reply:
No, if should match the module setting above. At the moment, node16 is nodenext, as no new resolution semantics have been introduced. Eventually maybe there will be a node24 if that version changes something.

Related, but "node" was renamed to "node10" for that reason.